### ISSUE TITLE
Don't put inflector rules in translation

### ISSUE THREAD

**Issue by vrabe** [getgrav/grav#2503]
Each language has its rules to form plural nouns from singular nouns. The rules in other languages can't 1-to-1 map to the rules in English. 

For example, plural forms of Chinese nouns are mostly the same as the singular. So inflectors are useless in Chinese. I don't know how to translate these inflector-related strings.

I also found a comment about this in Crowin:

> these pluralize settings cannot be translated into Romanian as we have very different grammar.
> most nouns are either male / female / neutral and their plurals are all over the place
>
> Google didn't manage to even begin to translate Romanian into English or vice-versa

I think the inflector rules should be separated from translation. These should not be translated.

**Comment by rhukster** [comment]
there are some languages where the inflector rules are valid though.

**Comment by vrabe** [comment]
Yes

The problem is Crowdin wants translators to translate every strings from English.

In `INFLECTOR_PLURALS` and `INFLECTOR_SINGULAR`, we can't change the keys, which means we can't change the matching rules. We can only change the replacement strings.

In `INFLECTOR_UNCOUNTABLE`, translators are asked to translate 'equipment', 'information', 'rice', ... to the target languages although these words may not be uncountable nouns in their languages.

So I think inflector rules are not appropriate to be translated in Crowdin. They should be put in somewhere else so that translators can edit these rules freely if they need inflector in their languages.

**Comment by mahagr** [comment]
I agree, most projects have a fixed file for plural etc rules, which cannot be translated but are defined as a separate rule file outside of the rest of the translations.
